ca/bootstrap.go

```diff
@@ -120,25 +118,21 @@ func BootstrapServer(ctx context.Context, token string, base *http.Server, optio
 		return nil, errors.New("server TLSConfig is already set")
 	}
 
-	client, err := Bootstrap(token)
+	b, err := createBootstrap(token)
 	if err != nil {
 		return nil, err
 	}
 
-	req, pk, err := CreateSignRequest(token)
-	if err != nil {
-		return nil, err
-	}
-
-	sign, err := client.Sign(req)
-	if err != nil {
-		return nil, err
+	// Make sure the tlsConfig have all supported roots on RootCAs.
```

Comment:
Make sue the tlsConfig `has` all supported roots on RootCAs. Change in this spot and in the one above.